Assets/DLC/Expansion2/Gameplay/XML/Text/pt_BR/CIV5GameTextInfos_Civilopedia_Expansion2.xml

```diff
@@ -89,8 +89,7 @@
 			<Text>O Colapso</Text>
 		</Row>
 		<Row Tag="TXT_KEY_CIV5_ASSYRIA_TEXT_8">
-			<Text>Após a morte de Assurbanipal em 627 a.C., vários sucessores rivais de curta duração reivindicaram o trono; A Assíria foi empurrada para um período de guerra civil. A desordem e a devastação trazidas por esses conflitos levaram diretamente ao declínio e à dissolução não apenas do Império Assírio, mas também da cultura distintamente identificada como "Assíria". Em 605 a.C., uma aliança de Babilônios e Medos derrotaria uma força Egípcia-Assíria em Carquemis e Assíria se tornaria uma província Babilônica. Após o colapso do Império Babilônico, a tradicional terra Assíria seria invadida por ondas de invasores que iam dos Persas aos Árabes. O conhecimento de suas glórias e obras virtualmente desapareceria até ser trazido à luz pelos arqueólogos e historiadores modernos, começando com as escavações da famosa Nínive em 1845 d.C..
-</Text>
+			<Text>Após a morte de Assurbanipal em 627 a.C., vários sucessores rivais de curta duração reivindicaram o trono; A Assíria foi empurrada para um período de guerra civil. A desordem e a devastação trazidas por esses conflitos levaram diretamente ao declínio e à dissolução não apenas do Império Assírio, mas também da cultura distintamente identificada como "Assíria". Em 605 a.C., uma aliança de Babilônios e Medos derrotaria uma força Egípcia-Assíria em Carquemis e Assíria se tornaria uma província Babilônica. Após o colapso do Império Babilônico, a tradicional terra Assíria seria invadida por ondas de invasores que iam dos Persas aos Árabes. O conhecimento de suas glórias e obras virtualmente desapareceria até ser trazido à luz pelos arqueólogos e historiadores modernos, começando com as escavações da famosa Nínive em 1845 d.C..</Text>
```

Comment:
Não faça isso.  Mudanças de formatação do texto não tem nenhum efeito no jogo, mas tornam mais difícil o trabalho de manutenção desses arquivos.  Se for possível reverta essa e todas as demais mudanças desse tipo.